src/_pytest/assertion/rewrite.py

```diff
@@ -1074,13 +1074,14 @@ def try_makedirs(cache_dir) -> bool:
 
 def get_cache_dir(file_path: Path) -> Path:
     """Returns the cache directory to write .pyc files for the given .py file path"""
-    if sys.version_info >= (3, 8) and sys.pycache_prefix:
+    # Type ignored until added in next mypy release.
```

Comment:
Would be good to link an issue in general, but ok for now.

Reply:
mypy can detect outdated `type: ignore`s so it's easy to get rid of when updating mypy.